jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -179,36 +173,29 @@
      * <tt>Conference</tt> instance is to be initialized
      * @param id the (unique) ID of the new instance to be initialized
      * @param conferenceName world readable name of this conference
-     * @param enableLogging whether logging should be enabled for this
      * {@link Conference} and its sub-components, and whether this conference
      * should be considered when generating statistics.
      * @param gid the optional "global" id of the conference.
      */
     public Conference(Videobridge videobridge,
                       String id,
-                      EntityBareJid conferenceName,
-                      boolean enableLogging,
+                      @NotNull EntityBareJid conferenceName,
```

Comment:
Now that I think about it, do we want to force this to be present? It's not essential, and non-jicofo controllers might not use it

Reply:
I thought we were relying on this for some things now? (though maybe that would still fall under just the Jicofo case).  @damencho ?

Reply:
I know he had originally made it a String, which I think would be nice, but we require having the domain and stuff in there so he did the EntityBareJid to enforce that.

Reply:
We require it to have a JID format if it's present. Otherwise, we only really use it if callstats is enabled. It feels wrong to require a JID even when the bridge doesn't use XMPP

Reply:
A name seems like it would be useful even in the non-Jicofo case (we use it in debug state and in log context), so it seems likely they'd fake an `EntityBareJid` anyway.  I really liked the String idea, but then we had to do all this validation on it since we _need_ it to be a valid `EntityBareJid`, so enforcing it in the type was better.

I agree an actual `EntityBareJid` is not always needed, but I think the question is would a non-Jicofo user be more likely to want to use `null`, or a fake `EntityBareJid` to get more context?

After looking into it some more, it looks like the only place the bridge cares about it being an _actual_ Jid is in `describeShallow`--and that already handles the case where the conversion may fail, so maybe we can just use a `String` at this layer?